packages/tao/src/commands/ngcli-adapter.ts

```diff
@@ -1003,3 +1009,23 @@ const getTargetLogger = (
   );
   return tslintExecutorLogger;
 };
+
+function splitProjectFileUpdatesFromWorkspaceUpdate(
+  host: Tree,
+  r: { eventPath: string; content: Buffer }
+) {
+  const workspace: {
```

Comment:
We have a type for `WorkspaceJsonConfiguration`

Reply:
Not applicable here since we are using the `configFilePath` property.